react/OrderPayment.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { PaymentMethod } from 'vtex.order-details'
+
+import { useOrder } from './components/OrderContext'
+import { useCurrency } from './components/CurrencyContext'
+
+const OrderPayment = () => {
+  const order = useOrder()
+  const currency = useCurrency()
+
+  const { payments, transactionId } = order.paymentData.transactions[0]
```

Comment:
if `paymentData` is `undefined` or `transactions` has no children what happens to `payments`?

Reply:
Same thing answered above. It's supposed to have them.